### PR TITLE
chore: upgrade prettier to v3

### PR DESCRIPTION
**Pull request by ixahmedxi**

### CODE REVIEW

**Review by juliusmarminge — DISMISSED** [review]

**Comment by juliusmarminge** [comment]
hmm typedoc isn't happy :/ 

CleanShot 2024-01-17 at 10 36 25@2x

**Comment by ixahmedxi** [comment]
> hmm typedoc isn't happy :/
> 
> CleanShot 2024-01-17 at 10 36 25@2x

I will investigate this

File: www/package.json [hunk 1]
```diff
@@ -75,13 +75,13 @@
...
     "dotenv": "^16.0.1",
-    "eslint": "^8.40.0",
+    "eslint": "^8.56.0",
     "next": "^14.0.1",
     "next-auth": "^4.22.1",
     "npm-run-all": "^4.1.5",
     "oauth": "^0.10.0",
     "postcss": "^8.4.14",
-    "prettier": "^2.8.8",
+    "prettier": "npm:prettier@^2.8.8",
```
Comment (KATT) [inline www/package.json:84]: I pinned this to prettier 2 -- @ixahmedxi see if we can just remove it here instead somehow. Idk how/why prettier is included in docusaurus' build

**Review by KATT — CHANGES_REQUESTED** [review]
Needs updating now that `@trpc/core` has been removed

Might be easiest to cherry-pick the `package.json`-changes to a new PR and let the autofixer do its thing

**Pull request closed without merging**